docs/configuration/traces-config.md

```diff
@@ -145,6 +145,26 @@ scrape_configs:
 # `update` only modifies an existing k/v and `insert` only appends if the k/v
 # is not present. `upsert` does both.
 [ prom_sd_operation_type: <string> | default = "upsert" ]
+# Configures what methods to use to do association between spans and pods.
+# PromSD processor matches the IP address of the metadata labels from the k8s API
+# with the IP address obtained from the specified pod association method.
+# If a match is found then the span is labeled.
+#
+# Options are `ip`, `net.host.ip`, `k8s.pod.ip`, `hostname` and `connection`.
+#   - `ip`, `net.host.ip` and `k8s.pod.ip`, `hostname` match spans tags.
+#   - `connection` inspects the context from the incoming requests (gRPC and HTTP).
```

Comment:
The connection association can be elaborated a bit in the docs. Its not obvious that it will look at the RemoteAddr in the requests.

Reply:
It also uses a different method to inspect the IP in gRPC. Is it worth mentioning? It's something that is done automatically. The user won't have access to this code.

Reply:
Yeah its fine to just mention.